source/opt/instruction.h

```diff
@@ -62,6 +62,9 @@ struct Operand {
   Operand(spv_operand_type_t t, std::vector<uint32_t>&& w)
       : type(t), words(std::move(w)) {}
 
+  Operand(spv_operand_type_t t, const std::vector<uint32_t>& w)
```

Comment:
Hmm... If you'd think it's necessary to keep a copy of `w` after passing it in this function, what about changing the above function to take `std::vector<uint32_t>`? That way, we only have one method to maintain, and the performance is on par:
* If passed in a rvalue, it will be moved into `w` and then moved to `words`. An extra move compared to this implementation.
* If passed in a lvalue, it will be copied to `w` and then moved to `words`. An extra move compared to this implementation.

But this is nit picking. Coder's choice.